src/components/CookieForm.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { Button } from 'components/html/Button'
+import { initialViewButtonStyle } from 'shared/styles/Styles'
+import RoutingPath from 'routes/RoutingPath'
+import styled from 'styled-components'
+
+export const CookieForm = () => {
+	const [acceptedCookies, setAcceptedCookies] = useState<string>('false')
+
+	const checkLocalStoreForAcceptedCookies = () => {
+		(localStorage.getItem('acceptedCookies') === 'true') ? setAcceptedCookies('true') : setAcceptedCookies('false')
+	}
+
+	const updateLocalStoreWithAcceptedCookies = () =>{
+		localStorage.setItem('acceptedCookies', 'true')
+		setAcceptedCookies('true')
+	}
+
+	useEffect(() => {
+		checkLocalStoreForAcceptedCookies()
+	}, [acceptedCookies])
+		
+
+	if (acceptedCookies === 'false') {
```

Comment:
Räcker med följande:
`if (acceptedCookies) {}`

Reply:
Changed to if (!accepedCookies) as discussed